tester/orb.yaml

```diff
@@ -134,3 +134,47 @@ jobs:
           command: |
             cd <<parameters.cwd>>
             python -m pytest <<parameters.args>>
+
+  integration-tests:
+    description:
+      Runs integration tests using docker-compose
+    docker:
+      - image: <<parameters.executor>>
+    resource_class: <<parameters.resource_class>>
+    parameters:
+      args:
+        default: ''
+        description: |
+          Arguments to pass to downstream container
+        type: string
+      cwd:
+        default: '.'
+        description: >
+          Working directory used when running docker-compose commands.
+        type: string
+      executor:
+        default: cimg/base:2020.01
+        description: >
+          Name of the docker image to use to execute the job.
+        type: string
+      resource_class:
+        default: small
+        type: string
+      service_name:
+        default: integration-tests
```

Comment:
Let's make this a required arg instead of having a default value here -- this is a bit more opinionated than we should be in an OSS context :)